Resources/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "bazinga-translator",
+  "version": "1.0.0",
+  "description": "A pretty nice way to use your translation messages generated by the BazingaJsTranslationBundle in your JavaScript.",
+  "keywords": [ "translation", "symfony" ],
+  "license": "MIT",
+  "author": {
+    "name": "William Durand",
+    "email": "will+git@drnd.me"
+  },
+  "contributors": [
+    {
+      "name": "Bruno Sampaio",
+      "email": "bens.sampaio@gmail.com"
+    }
+  ],
+  "files": [
+    "js",
```

Comment:
you should include only ``js/translator.js``. The folder includes a bunch of other files which are needed only by the testsuite